.github/workflows/deploy-pr-preview.yml

```
@@ -0,0 +1,22 @@
+name: Deploy pr preview
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - closed
+    paths:
+      - "docs/sources/**"
+
+jobs:
+  deploy-pr-preview:
+    uses: grafana/writers-toolkit/.github/workflows/deploy-preview.yml@robbymilo/deploy-preview
+    with:
+      sha: ${{ github.event.pull_request.head.sha }}
+      branch: ${{ github.ref_name }}
```

Same change as https://github.com/grafana/website/commit/17a0f7f6c3529dd9bc13911fb8332981e5aba128 (whitespace mistake notwithstanding)

```suggestion
      branch: ${{ github.head_ref }}
```